lockbox-ios/Presenter/AccountSettingPresenter.swift

```diff
@@ -29,6 +29,21 @@ class AccountSettingPresenter {
         }.asObserver()
     }()
 
+    lazy private(set) var unLinkAccountTapped: AnyObserver<Void> = {
```

Comment:
Now that you are using this tap observer, the `unlinkAccountObserver` can be `private` protection level.

Reply:
I don't think we can make this private cause this method is currently being called by the view. Perhaps am missing something . Please do tell

Reply:
Scratch this all is well.